test/libsolidity/syntaxTests/denominations/szabo_invalid.sol

```diff
@@ -0,0 +1,7 @@
+contract C {
+  function f() {
+    uint x = 1 szabo;
```

Comment:
I think it is fine not having a special message for these, they cannot be misused as it is a special syntax.

The problem of misleading code such as `1 * szabo` is already possible and we're not trying to fix it.